packages/core/addon/services/-scheduler.js

```diff
@@ -37,7 +40,9 @@ function typeResolver(type) {
       ? yield this.calumaOptions[methodName]?.(uncachedIdentifiers)
       : [];
 
-    const allResults = [...cached, ...(result?.toArray?.() ?? result ?? [])];
+    const allResults = toArrayIsDeprecated
+      ? [...cached, ...(result ?? [])]
+      : [...cached, ...(result?.toArray?.() ?? result ?? [])];
```

Comment:
In which situation would we still need the old syntax?

Reply:
When using `ember-caluma` with `ember-data < 4.7`.

Otherwise, we have to do a major version and specify `ember-data > 4.7` as a dependency.

Reply:
Would be interesting to know since when exactly the new syntax is supported - but I wasn't able to find it in a quick search. Thanks!